.circleci/config.yml

```diff
@@ -24,16 +24,6 @@ STUFF:
         - /\//
 
 jobs:
-  lint:
-    working_directory: /app
-    docker:
-      - *python_image
-    steps:
-      - checkout
-      - run: pip install yapf==0.16.1 && pip install flake8==3.5.0 && pip install pytest==3.6.3
-      - run: yapf --diff --recursive ./
-      - run: flake8 .
```

Comment:
linting tests shouldn't be removed from CI

Reply:
I updated this to match with lando-api. I could reimplement this with black + flake8 to match the updated tests. Is this not redundant since we are running linting as part of `test_py3_codequality`? (perhaps the latter should be removed?)